src/graphene-simd4x4f.h

```diff
@@ -68,8 +68,8 @@ GRAPHENE_BEGIN_DECLS
  *
  * Since: 1.0
  */
-GRAPHENE_VECTORCALL
 static inline graphene_simd4x4f_t
+GRAPHENE_VECTORCALL
```

Comment:
Could you put this on the same line as `static inline graphene_simd4x4f_t` instead of a separate line? It makes reading a bit simpler that way.